src/login/logind-dbus.c

```diff
@@ -984,6 +1002,48 @@ static int method_create_session(sd_bus_message *message, void *userdata, sd_bus
         return r;
 }
 
+static int method_create_session(sd_bus_message *message, void *userdata, sd_bus_error *error) {
+        const char *service, *type, *class, *cseat, *tty, *display, *remote_user, *remote_host, *desktop;
+        pid_t leader;
+        uid_t uid;
+        int remote;
+        uint32_t vtnr = 0;
+        int r;
+
+        assert(message);
+
+        assert_cc(sizeof(pid_t) == sizeof(uint32_t));
+        assert_cc(sizeof(uid_t) == sizeof(uint32_t));
+
+        r = sd_bus_message_read(message, "uusssssussbss",
+                                &uid, &leader, &service, &type, &class, &desktop, &cseat,
+                                &vtnr, &tty, &display, &remote, &remote_user, &remote_host);
+        if (r < 0)
+                return r;
+
+        return create_session(message, userdata, error,
+                              uid, leader, -EBADF /* pidfd= */, service, type, class, desktop, cseat, vtnr, tty, display, remote, remote_user, remote_host, 0 /* flags */);
```

Comment:
comment with the var name is usually before the value, also probably need more line breaks, usually one param per line

Reply:
When it comes to line breaks I think it would make sense to make this consistent with `sd_bus_message_read()` above.

Reply:
well, that sd_bus_message_read() line kinda the outlier here...